packages/react-server-test-pages/pages/bottleneck/elements.js

```diff
@@ -15,19 +15,25 @@ const IndigoThing = () => <div className="indigo-thing">This should be indigo</d
 const YellowThing = () => <div className="yellow-thing">This should be yellow</div>;
 const ColorWheel = [RedThing(), BlueThing(), PurpleThing(), GreenThing(),
 	OrangeThing(), IndigoThing(), YellowThing()];
+const ColorSize = ColorWheel.length;
 
+/**
+* This page is a smoke test to determine whether or not the number of elements in
+* a page is a performance bottleneck for react-server. It returns a huge assortment of
+* randomly generated color-coded elements. Metrics are created in the browser's console
+* related to performance metrics (see react-server.core.ClientController).
+*/
 export default class ElementsPage {
 
 	getElements() {
 
 		const colorThings = [];
-		for (var i = 0; i < 99999; i++) {
-			let selection = Math.floor(Math.random() * 7);
+		for (var i = 0; i < 10000; i++) {
```

Comment:
After changing this behavior such that I was correctly returning a large number of elements (not one element with a large number of children), I found myself running out of heap space in javascript, so this number came down intentionally.